### ISSUE TITLE
Angular 11 support

### ISSUE THREAD

**Issue by Nashima-ariff** [tanansatpal/ngx-mat-intl-tel-input#103]
I'm using Angular 11.2.1 and try to use a ngx - mat-Intl-tel-input. When I am installing the package cli shows it has peer dependencies are set to 9.0.0. 
And If I try to implement with angular 11 project the ngx-mat-tel-input is hidden in UI.
But when I click on those hidden element I have dropdown menu of country..

Has this package been compatible with angular 11.2.1?

![Screenshot 2021-03-29 224443](https://user-images.githubusercontent.com/53169779/112874340-72eada00-90e0-11eb-9c11-2394f75eb093.png)

this package is not compatible.

**Comment by nbaker82** [comment]
Yes, there is an issue with peer dependencies and the usage of ~ over ^. I have a pullRequest out to fix this issue just waiting on review.

**Comment by specie** [comment]
As a temporary solution, you can install the pull request with: npm install tanansatpal/ngx-mat-intl-tel-input#pull/98/head --save

**Comment by gertvv84** [comment]
Hello, any updates about this issue?

Thanks!

**Comment by danj1980** [comment]
Same issue here.
